src/vnsw/agent/vrouter/ksync/ksync_init.cc

```diff
@@ -92,31 +91,23 @@ void KSync::RegisterDBClients(DB *db) {
 void KSync::Init(bool create_vhost) {
     NetlinkInit();
     VRouterInterfaceSnapshot();
-    // TODO: JW-882: uncomment the code
-    /* InitFlowMem(); */
+    InitFlowMem();
     ResetVRouter(true);
-
-    /* On Windows, vhost interface is created when Hyper-V switch is created */
-    #ifndef _WIN32
     if (create_vhost) {
+        // On Windows, vhost interface is created when Hyper-V switch is
```

Comment:
I would revert this change, so that the whole condition is enclosed in compilation guard. This way we could explicitly show that Agent on Windows is not responsible for creating the vhost interface.